### ISSUE TITLE
tmux.conf does not work on Mac OS. 

### ISSUE THREAD

**Issue by ss7pro** [PiotrProkop/dotfiles#1]
Do you have plans to support this platform ?

**Comment by PiotrProkop** [comment]
Yes, this [commit](https://github.com/PiotrProkop/dotfiles/commit/880f7dc9875dfc3e0c2439c7346037e51f6888e7) adds a simplified tmux.conf. It was tested on MacOS :D
